### DISCUSSION TITLE
The fate of the attributes from the Karma Charger videos

### DISCUSSION THREAD

**Discussion by CDTH06** [nosoop/SM-TFCustomAttributeStarterPack#52] [Q&A] [answered]
After Karma left YouTube, I wondered what would happen to the attributes from his videos? Will you give access to these attributes for community, or they will "die" like Karma's channel?

**Comment by nosoop** [comment, marked as answer]
Thanks for asking &mdash; I've moved this question from [nosoop/SM-TFCustAttr](https://github.com/nosoop/SM-TFCustAttr) to a newly-added discussion section for [nosoop/SM-TFCustomAttributeStarterPack](https://github.com/nosoop/SM-TFCustomAttributeStarterPack).

Most of the attributes I've written for him are available in this repository (the one I've transferred this question to), as well as the legacy CW3 repository at [nosoop/SM-TFCustomWeaponAttributes](https://github.com/nosoop/SM-TFCustomWeaponAttributes).  The [attribute list](https://github.com/nosoop/SM-TFCustomAttributeStarterPack/wiki/Custom-Attribute-List) provides information on which attribute was created for which video, though obviously the links point to now-removed videos.

I currently have no intention of removing any code, aside from the plugins that have duplicated functionality (such as `mod crit type vs disguised players` being replaced with `mod crit type on target condition`, among others).

**Reply by CDTH06** [reply]
Ok. What about attributes from another Karma videos, which not included in Starter Pack? Will they be added? And do you planing to add some new attributes?

**Reply by nosoop** [reply]
Any specific ones you're looking for?  Keep in mind that there were other developers creating custom attributes as well, which I have no access to.

As I indicated in the main page of the repository:
> I don't run a Custom Weapon server, so most of the plugins are made through work-for-hire.

I currently do not have new attributes in the works.  You (or any other person) are welcome to contact me over AlliedModders to negotiate funding to work on new additions.

**Reply by nosoop** [reply]
(Make sure to also look over the custom buffs / lunchbox effects sections, as some plugins are implemented as a specific value of another attribute.)
